### PR TITLE
Ajuste no valor armazenado e method http

### PR DESCRIPTION
**Pull request by chapolin**

### CODE REVIEW

**Comment by rafaell-lycan** [comment]
Hello, @chapolin! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/rafaell-lycan/sabesp-mananciais-api/pulls/41). Please check our [documentation](https://ebertapp.io/docs) for more information.

File: lib/Sabesp.js [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
       request({
           'url': url,
-          'method': 'POST',
+          'method': 'GET',
```
Comment (rafaell-lycan) [inline lib/Sabesp.js:27]: Expected indentation of 8 spaces but found 10.

[Read more about it here.](http://eslint.org/docs/rules/indent)

**Review by wbruno — APPROVED** [review]

File: lib/Sabesp.js (hunk 1 above)
Comment (chapolin) [reply lib/Sabesp.js:27]: Precisa corrigir isso? Apenas troquei o valor de POST por GET :P

Comment (rafaell-lycan) [reply lib/Sabesp.js:27]: Isso é boot. Desculpa @chapolin

**Pull request merged**